src/main.rs

```diff
@@ -1,3 +1,23 @@
-fn main() {
-    println!("Hello, world!");
+pub mod collectors;
+
+use crate::collectors::register_collectors;
+use hyper::Server;
+use monitorbot::MetricProvider;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    dotenv::dotenv().ok();
+    let addr = "0.0.0.0:3001".parse().unwrap();
```

Comment:
Note that you're not following the `PORT` environment variable (which it'd be nice to rename to `MONITORBOT_PORT`).

Reply:
OK, so we want first try to get a port number from an env var `MONITORBOT_PORT`, if the env var is not set, we fall back to port `80`, correct?

Reply:
What I'd do is to keep a non-80 port by default, so that when run the server with `cargo run` locally it doesn't try to bind to the restricted port 80. Then, in the Docker container we set `MONITORBOT_PORT` to 80.

Still, we can fix this after the PR is merged.